directory/cli/src/user.py

```diff
@@ -74,7 +74,95 @@ def show(login):
             error = '{}: user not found'.format(login)
             raise click.ClickException(error)
 
-    wrapper_commands = [
+    if not utils.advanced_mode_enabled():
+        @user.command(help='Create a new user')
+        def create():
+            click.echo('Please enter details for the following:')
+
+            params = {
+                        'login': click.prompt('  Username'),
+                        'first': click.prompt('  First name'),
+                        'last': click.prompt('  Surname'),
+                        'email': click.prompt('  Email'),
+            }
+            if click.confirm(
+                'Would you like to set a specific UID for this user? (Default: Autogenerated UID)'
+            ):
+                params = { **params, 'uid': click.prompt('  UID') }
+
+            group_id = _get_group_id('clusterusers')
+
+            if group_id:
+                params = { **params, 'gidnumber': group_id[0] }
```

Comment:
I believe we discussed irl yesterday moving the indexing into `_get_group_id`, since it seems more logical to have this method just return the GID rather than a list containing only the GID - is this possible or does it need to live outside?

Reply:
We did, this and other some of the other comments have been addressed in #73 (The PR I mentioned would be coming soon)